changelogs/CHANGELOG-1.4.md

```diff
@@ -1,5 +1,307 @@
 # Changelog since v1.3.0
 
+## v1.4.1
+
+Date: 2022-11-8
+
+### Changes
+
+- Add 1.4.0 to readme compatibility matrix. (#480, @xinyanw409)
```

Comment:
Can you make PR and gitid hyperlink like in this one? https://github.com/kubernetes-csi/external-snapshotter/blob/master/CHANGELOG/CHANGELOG-6.1.md#api-change